svc-composition-service/app/config/constants.py

```diff
@@ -0,0 +1,77 @@
+#(C) Copyright [2022] American Megatrends International LLC
+#
+#Licensed under the Apache License, Version 2.0 (the "License"); you may
+#not use this file except in compliance with the License. You may obtain
+#a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#License for the specific language governing permissions and limitations
+# under the License.
+
+EVENT_DESTINATION_URL = "/ODIM/v1/EventService/CompositionServiceEvent"
```

Comment:
Events are handled from plugin to eventservice via messagebus, can you explain the need of this URI

Reply:
redis-tools has redis client that we just use to connect to in-memory and on-disk database services. We are not installing any redis database locally in the composition-service image.